### ISSUE TITLE
Funkmodul ohne Servicemeldungen

### ISSUE THREAD

**Issue by MathiasJZ** [jens-maus/RaspberryMatic#762]
Ich möchte hie mal nachfragen, ob man beim Funkmodul ebenfalls die Servicemeldungen abschalten kann,, wenn bei Gräten die Servicemeldungen deaktiviert sind.
Es ist schon nervig, dass (in meinem Fall) das Funbkmodul 11 Monate gelb-blau blinkt, weil Aktoren der Weihnachtsbeleuchtung fehlen.
Man weiß jetzt nicht, liegen wieder irgendwelche Kommunikationsstörungen vor, oder liegt das am fehlendem Aktor.
Gruß,
Mthias

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by jp112sdl** [comment]
> Mir würde es ja schon reichen wenn es einen "Fake-Aktor" geben würde, der dann deren Stelle einnimmt.

https://github.com/jp112sdl/HB-UNI-Sen-DUMMY-BEACON
Damit überbrücke ich die Winterzeit. 
Getestet mit Sensoren. Sollte aber auch mit Aktoren funktionieren.

**Comment by MathiasJZ** [comment]
Gut schaue ich mir später mal an.
Da das Teil 8 Sensoren unterstützt,
wäre es genau das richtige für mich.
2 von den Teilen gebaut einmal sensorseitig, einmal aktorseitig
Gruß,
Mathias

Am 18. Januar 2020 20:04:40 schrieb Jérôme <notifications@github.com>:

**Comment by jp112sdl** [comment]
> Gehen die auch bei den Asksin++

Ja, das spielt keine Rolle. Der Dummy sendet ja nur stellvertretend für ein anderes Gerät die entpsrechenden Funktelegramme
